war/src/main/js/sortable-drag-drop.js

```diff
@@ -23,6 +23,14 @@ function registerSortableDragDrop(e) {
       forceFallback: true, // Do not use html5 drag & drop behaviour because it does not work with autoscroll
       scroll: true,
       bubbleScroll: true,
+      onChoose: function (event) {
+        const draggableDiv = event.item;
+        const height = draggableDiv.clientHeight;
+        draggableDiv.style.height = `${height}px`;
```

Comment:
not compatible with IE 11 as is, this LTS line still supports it so would need to be adjusted like so:
```suggestion
        draggableDiv.style.height = height + 'px';
```